oktapam/data_source_team_settings.go

```diff
@@ -0,0 +1,45 @@
+package oktapam
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/okta/terraform-provider-oktapam/oktapam/client"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/attributes"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/descriptions"
+)
+
+func dataSourceTeamSettings() *schema.Resource {
+	return &schema.Resource{
```

Comment:
I believe you are going to add all other attributes for team settings resource too and mark those as computed.